Linux/build_openssl.sh

```diff
@@ -74,6 +74,10 @@ if [[ $# -gt 0 ]] && [[ $1 == "debug" || $2 == "debug" || $3 == "debug" || $4 ==
     ADDITIONAL_CONF="-g "
 fi
 
+if [[ $# -gt 0 ]] && [[ $1 == "no-threads" || $2 == "no-threads" || $3 == "no-threads" || $4 == "no-threads" ]] ; then
```

Comment:
After adding this option, the max input parameters should be 5. So this check and the above similar checks should be updated.

Reply:
changed to `if [[ "$*" == *"no-threads"* ]] ;`, also for `space-out` and `debug`